client/view/Form/form.js

```diff
@@ -86,3 +90,23 @@ async function updateStatsView(query) {
 
   return true
 }
+
+export function submitForm(query) {
+  if (query) {
+    textarea.value = query
+  }
+
+  form.dispatchEvent(new Event('submit', { cancelable: true }));
+}
+
+function initUrlControl() {
+  let urlParams = new URLSearchParams(window.location.search);
+
+  if (urlParams.get('query')) {
+    submitForm(urlParams.get('query'));
+  }
+}
+
```

Comment:
Do we need to add `popstate` event listener to send AJAX request on Back button click?

Reply:
I think it will be used in this way vary rarely. But we can add it later. Can i create an issue?

Reply:
What page will do in Back press right now?

Reply:
Form result doesn't change. And i told, that very few people will use this feature for it. 
This is more form give a link to the particular query.

Reply:
> Form result doesn't change

No, this is a bug for me.

We should not break web platform.